packages/x-data-grid/src/components/GridPagination.tsx

```diff
@@ -48,7 +48,8 @@ const defaultLabelDisplayedRows: WrappedLabelDisplayedRows = ({ from, to, count,
   if (!estimated) {
     return `${from}–${to} of ${count !== -1 ? count : `more than ${to}`}`;
   }
-  return `${from}–${to} of ${count !== -1 ? count : `more than ${estimated > to ? estimated : to}`}`;
+  const estimateLabel = estimated && estimated > to ? `around ${estimated}` : `more than ${to}`;
```

Comment:
changed this since it is not always `more than` (we are estimating). If `to` is bigger, then it is definitely `more than`

Reply:
It looks better than the previous to me, thanks for doing this change.
Maybe reflecting it in the callout above [this](https://mui.com/x/react-data-grid/pagination/#cursor-based-pagination) section would also make sense?

Reply:
Done